.drone.yml

```diff
@@ -1,7 +1,26 @@
+---
 kind: pipeline
+type: kubernetes
 name: default
+
+tolerations:
+  - effect: NoSchedule
+    key: app
+    operator: Exists
+node_selector:
+  app: drone
+
+service_account_name: drone-rps-system-engineering
```

Comment:
Currently use a 1:1 mapping between service accounts and IAM roles. I would suggest you create a new service account and IAM role for this repo.